pkg/controllers/state/suite_test.go

```diff
@@ -1263,6 +1263,15 @@ var _ = Describe("Cluster State Sync", func() {
 		}
 		Expect(cluster.Synced(ctx)).To(BeFalse())
 	})
+	It("shouldn't consider the cluster state synced if a nodeclaim isn't launched", func() {
```

Comment:
Should we have a test case that validates that when we call the UpdateNodeClaim() call directly (rather than calling the reconcile method, which would be triggered by controller-runtime), we mark the cluster state as not synced?